zone/mob_movement_manager.cpp

```diff
@@ -936,6 +1078,16 @@ void MobMovementManager::UpdatePath(Mob *who, float x, float y, float z, MobMove
 	else if (who->IsUnderwaterOnly()) {
 		UpdatePathUnderwater(who, x, y, z, mob_movement_mode);
 	}
+	// If we can fly, and we have a target and we have LoS, simply fly to them.
+	// if we ever lose LoS we go back to mesh run mode.
+	else if (target && who->GetFlyMode() == GravityBehavior::Flying &&
```

Comment:
Formatting should be cleaned up but I'm still going to push this through

Reply:
@Akkadius Will fix formatting and push.  Is it  just the closing brace is indented too far?